lms/djangoapps/support/static/support/js/views/enrollment.js

```diff
@@ -39,12 +39,7 @@
                     user: user,
                     enrollments: this.enrollments,
                     formatDate: function(date) {
-                        if (!date) {
-                            return 'N/A';
-                        }
-                        else {
-                            return moment(date).format('MM/DD/YYYY (H:MM UTC)');
-                        }
+                        return date ? moment.utc(date).format('lll z') : 'N/A';
```

Comment:
I don't know what the format should be, keeping in mind that this tool was made after the course detail page. 

@sstack22 Are you okay with this change ?